Status: 2023q3: GitLab: corrections, changes (#276)
freebsd/freebsd-doc: website/content/en/status/report-2023-07-2023-09/gitlab.adoc

=== GitLab 16.3 Available

Link: +
link:https://about.gitlab.com/releases/2023/08/22/gitlab-16-3-released/[Gitlab 16.3 New Features] URL: link:https://about.gitlab.com/releases/2023/08/22/gitlab-16-3-released/[]

Contact: Matthias Fechner <mfechner@FreeBSD.org>

GitLab is a DevOps platform.
It brings velocity with confidence, security without sacrifice, and visibility into DevOps success.

Version 16.3 is now available on FreeBSD: please check the package:www/gitlab-ce[] port.
The upgrade is very important as version 16.3 will be required for all further upgrades.

> You cannot upgrade to 16.4 if you have not upgraded to 16.3.

Reviewer: Was the word "before" wrong? I found it clearer before. Maybe

```suggestion
You cannot upgrade to 16.4 directly: you need to have upgraded to 16.3 before.
```
Or

```suggestion
Upgrade to 16.4 is only possible from GitLab 16.3.
```

Author: Thanks @lsalvadore that _is_ better (at least, for me).  

----

_Thinking aloud_

It was the word 'before' that was, somehow, not _quite_ right, with my first speed-read of the report. Maybe not quite right as the ending to a sentence/paragraph.

Originally, I had to look three or more times at the paragraph, partly because of '16.3 will be required for _all_ further upgrades' in the second sentence.

Now, I imagine this as the third and final sentence (although there's no real need for further change): 

> Upgrading to 16.4 or greater will be impossible without first upgrading to 16.3.

– or: 

> 16.3 is a prerequisite to upgrades to 16.4 or greater.

<https://english.stackexchange.com/a/16437/11504> re: 'a prerequisite to'.
